vsts-api-patcher/src/main.rs

```diff
@@ -481,6 +483,260 @@ impl Patcher {
         }
     }
 
+    fn patch_operation_status_in_releases_query_parameters(
+        &mut self,
+        key: &[&str],
+        _value: &JsonValue,
+    ) -> Option<JsonValue> {
+        // Only applies to pipelines specs
```

Comment:
incorrect comment: should say "release specs"

Reply:
Updated in the latest commit.